mpt/impl/imap-mailbox/jcr/src/test/java/org/apache/james/mpt/imapmailbox/jcr/host/JCRHostSystem.java

```diff
@@ -129,8 +130,7 @@ public void resetUserMetaData() throws Exception {
         dir.mkdirs();
     }
 
-
-    @Override
+    //JCR tests are broken partly because of that method not being run 
```

Comment:
still valid?

Reply:
I forgot that comment, yes, still true but I have to fix it.

Reply:
Ok, I just put that comment here because I was wondering why the testsuite is disabled but didn't disable it in that commit. I don't expect to prevent the merge for that.